src/components/tx/TxRow.js

```diff
@@ -57,8 +39,10 @@ const TxRow = ({ tx }) => {
 
   const renderNewUi = () => (
     <tr onClick={_e => handleClickTr(tx.tx_id)}>
-      <td className=" d-lg-table-cell pe-3">{renderIdCell(tx.tx_id)}</td>
-      <td className=" d-lg-table-cell pe-3">{parseTimestamp(tx.timestamp)}</td>
+      <td className=" d-lg-table-cell pe-3">{ellipsis ? renderIdCell(tx.tx_id) : tx.tx_id}</td>
```

Comment:
Maybe we should use the EllipsiCell component here as well

Reply:
Agreed. Fixed as sugested.